runtime/commonMain/src/kotlinx/serialization/Migrations.kt

```diff
@@ -149,5 +159,85 @@ public fun <T : Any> SerialModule.getContextualOrDefault(value: T): KSerializer<
 @Suppress("UNUSED", "DeprecatedCallableAddReplaceWith")
 @Deprecated(
     message = "Top-level polymorphic descriptor is deprecated, use descriptor from the instance of PolymorphicSerializer or" +
-            "check for descriptor kind instead", level = DeprecationLevel.ERROR)
-public val PolymorphicClassDescriptor: SerialDescriptor get() = error("This property is no longer supported")
\ No newline at end of file
+            "check for descriptor kind instead", level = DeprecationLevel.ERROR
+)
+public val PolymorphicClassDescriptor: SerialDescriptor
+    get() = error("This property is no longer supported")
+
+@Deprecated(
+    "Deprecated for removal since it is indistinguishable from SerialFormat interface. " +
+            "Use SerialFormat instead.", ReplaceWith("SerialFormat"), DeprecationLevel.ERROR
+)
+public abstract class AbstractSerialFormat(override val context: SerialModule) : SerialFormat
+
+@Deprecated(
+    "This method was renamed to encodeToString during serialization 1.0 stabilization",
+    ReplaceWith("encodeToString<T>(value)"), DeprecationLevel.ERROR
+)
+public fun <T : Any> StringFormat.stringify(value: T): String = noImpl()
+
+@Deprecated(
+    "This method was renamed to decodeFromString during serialization 1.0 stabilization",
+    ReplaceWith("decodeFromString<T>(string)"), DeprecationLevel.ERROR
+)
+public fun <T : Any> StringFormat.parse(string: String): T = noImpl()
+
+@Deprecated(
+    "This method was renamed to encodeToString during serialization 1.0 stabilization",
+    ReplaceWith("encodeToString<T>(serializer, value)"), DeprecationLevel.ERROR
```

Comment:
Type argument seems unnecessary for all two-arguments invocations. Does IDEA automatically remove it?

Reply:
This particular replacement doesn't work without explicit type argument for some reason